.github/workflows/nginx/ssl/.placeholder

```diff
@@ -0,0 +1 @@
+# this file keeps the directory around for tests
```

Comment:
Can this be an empty `.gitignore`?

Reply:
I force-pushed a .gitignore. The rest is the same